src/Uno.UWP/Media/Playback/MediaPlayer.net.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Media.Playback
+{
+	public partial class MediaPlayer
+	{
+		public void Stop() { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)